kong/plugins/statsd/statsd_logger.lua

```diff
@@ -23,13 +23,46 @@ local stat_types = {
 }
 
 
-local function create_statsd_message(prefix, stat, delta, kind, sample_rate)
+local function create_statsd_message(prefix, stat, delta, kind, sample_rate, tags, tag)
   local rate = ""
   if sample_rate and sample_rate ~= 1 then
     rate = "|@" .. sample_rate
   end
 
-  return fmt("%s.%s:%s|%s%s", prefix, stat, delta, kind, rate)
+  if tag == nil or tags == nil then
+    return fmt("%s.%s:%s|%s%s", prefix, stat, delta, kind, rate)
+  end
+  
+  local metrics = {}
+  if tag == "dogstatsd" then
```

Comment:
I'd like to see a link pointing to the respective tag style specification.

Reply:
sure